packages/flutter/test/painting/text_painter_test.dart

```diff
@@ -1143,7 +1175,14 @@ void main() {
       e.toString(),
       isNot(contains('TextPainter.paint called when text geometry was not yet calculated')),
     );
+    painter.dispose();
   }, skip: isBrowser && !isCanvasKit); // https://github.com/flutter/flutter/issues/56308
+
+  test('TextPainter - debugDisposed', () {
+    final TextPainter painter = TextPainter();
```

Comment:
Check that at this point `debugDispose` is `false`?

Reply:
Done